repair/row_level.cc

```diff
@@ -2686,3 +2720,10 @@ future<> repair_service::stop() {
 repair_service::~repair_service() {
     assert(_stopped);
 }
+
+namespace debug {
+namespace repair {
+    static thread_local repair_meta_tracker* repair_meta_for_masters = &::_repair_meta_for_masters;
+    static thread_local repair_meta_tracker* repair_meta_for_followers = &::_repair_meta_for_followers;
+}
```

Comment:
I think these pointers are unnecessary, you can just wrap the `_repair_meta_for_masters` and `_repair_meta_for_followers` in `namespace debug::repair { }` and it will be enough.

Reply:
I wanted to avoid forward declarations for  _repair_meta_for_masters and _repair_meta_for_followers. Yes, I think it is better to put them inside debug namespace.

Reply:
I've sent a new version.